### PR TITLE
feat: [contentwarehouse] new module for contentwarehouse

### PR DESCRIPTION
**Pull request by ddixit14**
- feat: initial generation of contentwarehouse
- build: add the contentwarehouse module to monorepo

### CODE REVIEW

File: java-contentwarehouse/README.md [hunk 1]
```diff
@@ -0,0 +1,201 @@
+# Google Document AI Warehouse Client for Java
+
+Java idiomatic client for [Document AI Warehouse][product-docs].
```
Comment (suztomo) [inline java-contentwarehouse/README.md:3]: Where is product doc?

Comment (ddixit14) [reply java-contentwarehouse/README.md:3]: I think it is missing [here](https://github.com/googleapis/google-cloud-java/pull/8563/files#diff-74dbab4bda95728466e773ea1936a5ad82ce62ecac67f992a5554b1720ff8e2e) as well. The command I ran to generate this library is : 
`python3.9 new-client.py generate \
  --api_shortname=contentwarehouse \
  --proto-path=google/cloud/contentwarehouse \
  --name-pretty="Document AI Warehouse" \
  --product-docs="https://cloud.google.com/document-warehouse/docs/overview" \
  --api-description="Document AI Warehouse is an integrated cloud-native GCP platform to store, search, organize, govern and analyze documents and their structured metadata." \
  --release-level=preview \
  --transport=grpc \
  --language=java \
  --monorepo-url="https://github.com/googleapis/google-cloud-java"`

**Comment by ddixit14** [comment]
Unit tests failing with error message: `Error:  Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.10.1:compile (default-compile) on project proto-google-cloud-contentwarehouse-v1: Compilation failure: Compilation failure: 
Error:  /home/runner/work/google-cloud-java/google-cloud-java/java-contentwarehouse/proto-google-cloud-contentwarehouse-v1/src/main/java/com/google/cloud/contentwarehouse/v1/Document.java:[588,40] package com.google.cloud.documentai.v1 does not exist
Error:  /home/runner/work/google-cloud-java/google-cloud-java/java-contentwarehouse/proto-google-cloud-contentwarehouse-v1/src/main/java/com/google/cloud/contentwarehouse/v1/Document.java:[604,40] package com.google.cloud.documentai.v1 does not exist...`


image

Investigating this now.

File: java-contentwarehouse/README.md (hunk 1 above)
Comment (suztomo) [reply java-contentwarehouse/README.md:3]: Sorry, I found that the product doc is the bottom of this markdown file. (This is a style of how to write links)

https://screenshot.googleplex.com/98DoTbwLPnLitNh

**Comment by suztomo** [comment]
Do you see package `com.google.cloud.documentai.v1` somewhere in this monorepo? If so, the new module requires special dependency setup.

**Comment by ddixit14** [comment]
> Do you see package `com.google.cloud.documentai.v1` somewhere in this monorepo? If so, the new module requires special dependency setup.

Working on this now.

**Comment by ddixit14** [comment]
The tests are now failing with this error:
```
Error:  Tests run: 22, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 1.208 s <<< FAILURE! - in com.google.cloud.contentwarehouse.v1.DocumentServiceClientHttpJsonTest
Error:  com.google.cloud.contentwarehouse.v1.DocumentServiceClientHttpJsonTest.updateDocumentTest  Time elapsed: 0.011 s  <<< ERROR!
com.google.api.gax.rpc.NotFoundException: 
com.google.api.client.http.HttpResponseException: 404
POST [https://contentwarehouse.googleapis.com:443/v1/](https://contentwarehouse.googleapis.com/v1/)
Method not found for path 'https:///v1/'
	at com.google.api.gax.rpc.ApiExceptionFactory.createException(ApiExceptionFactory.java:90)
	at com.google.api.gax.rpc.ApiExceptionFactory.createException(ApiExceptionFactory.java:41)
	at com.google.api.gax.httpjson.HttpJsonApiExceptionFactory.createApiException(HttpJsonApiExceptionFactory.java:75)
	at com.google.api.gax.httpjson.HttpJsonApiExceptionFactory.create(HttpJsonApiExceptionFactory.java:54)
	at com.google.api.gax.httpjson.HttpJsonExceptionCallable$ExceptionTransformingFuture.onFailure(HttpJsonExceptionCallable.java:97)
	at com.google.api.core.ApiFutures$1.onFailure(ApiFutures.java:67)
	at com.google.common.util.concurrent.Futures$CallbackListener.run(Futures.java:1132)
	at com.google.common.util.concurrent.DirectExecutor...
```
image

**Pull request closed without merging**